apps/web/app/Guards/RouteGuard.tsx

```diff
@@ -32,8 +45,16 @@ export const checkRouteAccess = (path: string, userRole: UserRole) => {
       return segment === pathSegments[index];
     });
 
-    if (matches && roles.includes(userRole)) {
-      return true;
+    if (isLoggedIn) {
+      if (matches && roles.includes(userRole)) {
```

Comment:
Is the check for `roles.includes` necessary here? I believe this case is handled above? What do you think?